### PR TITLE
VPN-3205 - multihop re-add

### PR DESCRIPTION
**Pull request by mozrokafor**
## Description

  Fix commented out multihop functional tests

## Reference

    https://mozilla-hub.atlassian.net/browse/VPN-3205

[VPN-3205]: https://mozilla-hub.atlassian.net/browse/VPN-3205?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/functional/testMultihop.js [hunk 1]
```diff
@@ -2,639 +2,690 @@
...
-    currentCity = await vpn.getMozillaProperty(
-        'Mozilla.VPN', 'VPNCurrentServer', 'exitCityName');
-
-    for (let server of servers) {
-      if (currentCountryCode === server.code) {
-        for (let city of server.cities) {
-          if (city.name == currentCity) {
-            currentCity = city.localizedName;
-            break;
+ const assert = require('assert');
```
Comment (bakulf) [inline tests/functional/testMultihop.js:5]: why this extra space before each line? It makes almost impossible to review this PR. Can you please format this file correctly?

File: tests/functional/testMultihop.js [hunk 2]
```diff
@@ -6,25 +6,14 @@ const assert = require('assert');
...
-*/
-
 describe('Server list', function() {
   let servers;
   let currentCountryCode;
   let currentCity;
+  let currentCountry;
+
+  this.timeout(240000);
+  this.ctx.authenticationNeeded = true;
```
Comment (birdsarah) [inline tests/functional/testMultihop.js:16]: Why is this duplicating the next two lines?

File: tests/functional/testMultihop.js [hunk 3]
```diff
@@ -54,35 +43,32 @@ describe('Server list', function() {
...
-       await vpn.waitForElementAndClick(
-           homeScreen.selectMultiHopServerView.VPN_MULTHOP_CHEVRON)
-       assert(await vpn.getElementProperty(
-           homeScreen.selectMultiHopServerView.VPN_COLLAPSIBLE_CARD,
-     'expanded'))
-     });
-  */
-
-  it.only('check the countries and cities for multihop entries', async () => {
+  it.only('opening the entry and exit server list', async () => {
```
Comment (birdsarah) [inline tests/functional/testMultihop.js:46]: `it.only` means only this test will run in this test suite.

File: tests/functional/testMultihop.js [hunk 4]
```diff
@@ -117,524 +102,501 @@ describe('Server list', function() {
...
+        await vpn.waitForQuery(cityId.visible().prop(
+            'checked',
+            currentCountryCode === server.code &&
+                currentCity === city.localizedName));
+      }
+    }
+  });
+
+  // faulty on wasm -- fast follow to investigate
+  //  it.skip('Pick cities for entries', async () => {
```
Comment (birdsarah) [inline tests/functional/testMultihop.js:150]: why is this commented out? why not use `it.skip`?

File: tests/functional/testMultihop.js [hunk 5]
```diff
@@ -117,524 +102,501 @@ describe('Server list', function() {
...
+        console.log('  Waiting for cityId:', cityId);
+        await vpn.waitForQuery(cityId.visible().prop(
+            'checked',
+            currentCountryCode === server.code &&
+                currentCity === city.localizedName));
+      }
+    }
+  });
+
+  // faulty on wasm -- fast follow to investigate
```
Comment (birdsarah) [inline tests/functional/testMultihop.js:149]: but it has been working? (if the tests are currently green then I don't understand why it's faulty on wasm)

**Review by birdsarah — CHANGES_REQUESTED** [review]

**Comment by lesleyjanenorton** [comment]
Closing for staleness. Feel free to reopen if there is further work to commit.

**Pull request closed without merging**